Games/League of Legends/BETA Client/script.js

```diff
@@ -1,32 +1,61 @@
 include(["Functions", "QuickScript", "CustomInstallerScript"]);
+include(["Functions", "Shortcuts","Wine"]);
 include(["Functions", "Verbs", "sandbox"]);
+include(["Functions", "Verbs", "d3dx9"]);
 include(["Functions", "Verbs", "vcrun2015"]);
 
+// Installs League of Legends and creates two shortcuts.
+//
+// One shortcut called League of Legends which starts the launcher
+// and one shortcut called BETA Client which starts the BETA client
+//
+// Important: 
+// The BETA-Client has to be installed using the launcher.
+// After the upgrade the BETA Client will do the game patching.
+
 new CustomInstallerScript()
     .name("League of Legends BETA Client")
```

Comment:
What's missing here is the information that this script installs the EUW client. 

I believe that we want to support the other region clients too.
To do this, I don't think that we want to add an install script for every region. 
I believe it is better to ask the user during the installation, which region client he wants to install. 
This can be done the same way as with the language selection in the [Internet Explorer 7](https://github.com/PlayOnLinux/Scripts/blob/master/Internet/Internet%20Explorer%207.0/Online/script.js) script.